Source/SpectrumAnalysis/KTSequentialTrackFinder.cc

```diff
@@ -115,13 +121,33 @@ namespace Katydid
         }
         if (node->has("apply-power-cut"))
         {
-            SetApplyPowerCut(node->get_value("apply-power-cut", GetApplyPowerCut()));
-            SetPowerThreshold(node->get_value("power-threshold", GetPowerThreshold()));
+            SetApplyTotalPowerCut(node->get_value("apply-total-power-cut", GetApplyTotalPowerCut()));
+            SetTotalPowerThreshold(node->get_value("total-power-threshold", GetTotalPowerThreshold()));
         }
         if (node->has("apply-power-density-cut"))
         {
-            SetApplyDensityCut(node->get_value("apply-power-density-cut", GetApplyDensityCut()));
-            SetDensityThreshold(node->get_value("power-density-threshold", GetDensityThreshold()));
+            SetApplyAveragePowerCut(node->get_value("apply-average-power-cut", GetApplyAveragePowerCut()));
+            SetAveragePowerThreshold(node->get_value("average-power-threshold", GetAveragePowerThreshold()));
+        }
+        if (node->has("apply-total-snr-cut"))
+        {
+            SetApplyTotalSNRCut(node->get_value("apply-total-snr-cut", GetApplyTotalSNRCut()));
+            SetTotalSNRThreshold(node->get_value("total-snr-threshold", GetTotalSNRThreshold()));
+        }
+        if (node->has("apply-average-snr-cut"))
+        {
+            SetApplyAverageSNRCut(node->get_value("apply-average-snr-cut", GetApplyAverageSNRCut()));
+            SetAverageSNRThreshold(node->get_value("average-snr-threshold", GetAverageSNRThreshold()));
+        }
+        if (node->has("apply-total-nup-cut"))
+        {
+            SetApplyTotalUnitlessResidualCut(node->get_value("apply-total-nup-cut", GetApplyTotalUnitlessResidualCut()));
+            SetTotalUnitlessResidualThreshold(node->get_value("total-nup-threshold", GetTotalUnitlessResidualThreshold()));
+        }
+        if (node->has("apply-average-nup-cut"))
+        {
+            SetApplyAverageUnitlessResidualCut(node->get_value("apply-average-nup-cut", GetApplyAverageUnitlessResidualCut()));
+            SetAverageUnitlessResidualThreshold(node->get_value("average-nup-threshold", GetAverageUnitlessResidualThreshold()));
         }
         if (node->has("n-slope-points"))
         {
```

Comment:
For consistency with other processors, the strings of `slope-method` should use hyphens instead of underscores (even though the enum uses underscores).